data/news/multicore/multicore-2020-03.md

```diff
@@ -29,7 +29,7 @@ Onto the details! The various ongoing and completed tasks for Multicore OCaml ar
 * [ocaml-multicore/ocaml-multicore#240](https://github.com/ocaml-multicore/ocaml-multicore/pull/240)
   Proposed implementation of threads in terms of Domain and Atomic
 
-  A new implementation of the `Threads` library for use with the new `Domain` and `Atomic` modules in Multicore OCaml has been proposed. This builds Dune 2.4.0 which in turn makes it useful to build other packages. This PR is open for review.
+  A new implementation of the `Threads` library for use with the new `Domain` and [`Atomic`](/api/Atomic.html) modules in Multicore OCaml has been proposed. This builds Dune 2.4.0 which in turn makes it useful to build other packages. This PR is open for review.
```

Comment:
```suggestion
  A new implementation of the `Threads` library (for use with the new `Domain` and [`Atomic`](/api/Atomic.html) modules in Multicore OCaml) has been proposed. This builds Dune 2.4.0, which in turn makes it useful to build other packages. This PR is open for review.
```
Alternatively:
  A new implementation of the `Threads` library, which will be used with the new `Domain` and [`Atomic`](/api/Atomic.html) modules in Multicore OCaml, has been proposed. This builds Dune 2.4.0, which in turn makes it useful to build other packages. This PR is open for review.

Reply:
Looks good! Thank you for reviewing :)